packages/sui-deploy/bin/sui-deploy-spa.js

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env node
+/* eslint no-console:0 no-template-curly-in-string:0 */
+
+const program = require('commander')
+const NowClient = require('now-client')
+const { getSpawnPromise } = require('@schibstedspain/sui-helpers/cli')
+const {writeFile, removeFile} = require('@schibstedspain/sui-helpers/file')
+const BUILD_FOLDER = './public'
+const writePackageJson = (name) => writeFile(pkgFilePath, `{
+  "name": "@sui-deploy/${name}",
+  "scripts": {
+    "start": "serve . --single"
+  },
+  "dependencies": {
+    "serve": "latest"
+  }
+}`)
+
+program
+  .command('sui-deploy spa <name> <folder>')
+  .on('--help', () => {
+    console.log('  Description:')
+    console.log('')
+    console.log('  Deploy build folder to now.sh')
+    console.log('')
+    console.log('  Examples:')
+    console.log('')
+    console.log('    $ sui-deploy spa my-app-name ./public')
+    console.log('    $ NOW_TOKEN=my-token; sui-deploy spa my-app-name ./public')
+    console.log('')
+  })
+  .parse(process.argv)
+
+if (!process.argv.slice(2).length) {
+  program.outputHelp()
+}
+
+if (!process.env.NOW_TOKEN) {
+  console.log('ERR: NOW_TOKEN env variable is missing')
+  process.exit(1)
+}
+
+const [,, deployName, folder = BUILD_FOLDER] = process.argv
+const now = new NowClient(process.env.NOW_TOKEN)
```

Comment:
Are There a way to avoid this spawn, or the NowClient ?!

Now, you are mixing the now binary install in your system and the now cliente library. I thing that could be great, is have now install in your system is not a requirement to use this package.

What do you think ?!

Reply:
We can't: https://github.com/SUI-Components/sui/pull/102#issuecomment-326507914